src/components/App.jsx

```diff
@@ -25,33 +26,23 @@ class App extends Component{
   }
 
   render() {
-    const user = loadState('jwtAuth')
     const PrivateRoute = ({ component: ImportResourceTemplate, ...rest }) => (
       <Route
         {...rest}
         render={props =>
-          (user !== undefined && user.isAuthenticated) ? (
-            <ImportResourceTemplate {...props} />
-          ) : (
-            <Redirect
-              to={{
-                pathname: "/login",
-                state: { from: props.location }
-              }}
-            />
-          )
+          (this.props.currentSession) ? (<ImportResourceTemplate {...props} />) : null
         }
       />
     )
 
     return(
       <div id="app">
+        <LoginPanel />
```

Comment:
pulled the `LoginPanel` up to the top level.  it's what's responsible for attempting to reload a user on initial page load, so this makes sure that component runs regardless of the URL initially requested within the app (pulling the component up in this way was the easiest way to make sure that behavior always happens).

as a bonus, it also means that it'll be easier to ask a user to re-login while staying on the same page, if we want to wire things up so that happens if a user tries to do something that requires authN, but they no longer have a valid or refreshable session for some reason.